streams/src/main/java/org/apache/kafka/streams/processor/PartitionGrouper.java

```diff
@@ -28,9 +28,9 @@
     /**
      * Returns a map of task ids to groups of partitions.
      *
-     * @param topicGroups The subscribed topic groups
+     * @param topicGroups The map from the topic group id to topics
```

Comment:
We do not have any explanations about "topic group" and how it is generated from "source groups" in topology, and how it could be map to "partition groups" (i.e. there are some restrictions when user customize it, some of them explained above in DefaultPartitionGrouper). Could we also add those descriptions?